Arrowgene.Ddon.GameServer/Handler/InstanceEnemyKillHandler.cs

```diff
@@ -32,6 +37,14 @@ public override void Handle(GameClient client, StructurePacket<C2SInstanceEnemyK
             StageId stageId = StageId.FromStageLayoutId(layoutId);
             ushort subGroupId = 0;
 
+            // The training room uses special handling to produce enemies that don't exist in the QuestState or InstanceEnemyManager.
+            // Return an empty response here to not break the rest of the handling.
+            if (_ignoreKillsInStageIds.Contains(stageId.Id))
```

Comment:
The enemies in the training room should use the named params "Training" which already specify that the exp drop is 0%, this is probably not needed